iceoryx2-bb/cxx/tests/cmake/googletest.cmake

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
I thought we create the following scheme:
* `iceoryx2-bb/container/cxx/`
* `iceoryx2-bb/container/rust/`
* `iceoryx2-bb/container/python/` (in the future)

All other crates most likely will not be ported to any other language.

Reply:
I don't have strong feeling about the hierarchy here. I mostly tried to mirror the layout of `iceoryx2-ffi` here, but that may not be a good idea.